ERCS/erc-7846.md

```diff
@@ -0,0 +1,233 @@
+--- 
+eip: 7846
+title: Wallet Connection API  
+description: Adds JSON-RPC method for requesting wallet connection with modular capabilities.  
+author: Conner Swenberg (@ilikesymmetry), Jake Moxey (@jxom), Lukas Rosario (@lukasrosario).
+discussions-to: https://ethereum-magicians.org/t/erc-7846-wallet-connection-api/22245
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-12-15
+---
+
+## Abstract
+
+This ERC introduces a new wallet connection JSON-RPC method focused on extensibility. It leverages the modular capabilities approach defined in [ERC-5792](https://eips.ethereum.org/EIPS/eip-5792#wallet_getcapabilities) to streamline connections and authentication into a single interaction.
+
+## Motivation
+
+With applications beginning to require support for more sophisticated functionality in wallet connection flows, the need for a unified and extensible wallet connection JSON-RPC method has become more apparent.
+
+This is especially evident in the case of attempting to batch connection with authentication, where existing methods like `eth_requestAccounts` and `personal_sign` lack extensibility and require at least two separate user interactions (ie. connect and then sign).
+
+## Specification
+
```

Comment:
Since you're using the uppercase keywords (MUST), you should include the paragraph explaining them from the EIP template.